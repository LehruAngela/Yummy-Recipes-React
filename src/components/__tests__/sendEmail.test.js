import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import SendEmail from '../sendEmail';

describe('SendEmail', () => {
    const wrapper = shallow(<SendEmail />);
    const preventDefault = jest.fn();

    it('renders properly', () => {
        expect(shallowToJson(wrapper)).toMatchSnapshot();
      }); 

    it('handles SendEmail', () => {
        expect(wrapper.instance().handleSendEmail({ preventDefault }));
    });

 
});