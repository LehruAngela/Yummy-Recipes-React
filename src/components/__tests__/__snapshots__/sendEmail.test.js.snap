// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SendEmail renders properly 1`] = `
<div>
  <div
    aria-hidden="true"
    aria-labelledby="exampleModalLabel"
    class="modal fade"
    id="exampleModal"
    role="dialog"
    tabindex="-1"
  >
    <div
      class="modal-dialog"
      role="document"
    >
      <div
        class="modal-content"
      >
        <div
          class="modal-header"
        >
          <h4
            class="modal-title"
            id="exampleModalLabel"
          >
            Reset Password
          </h4>
          <button
            aria-label="Close"
            class="close"
            data-dismiss="modal"
            id="closeModal"
            type="button"
          >
            <span
              aria-hidden="true"
            >
              Ã—
            </span>
          </button>
        </div>
        <div
          class="modal-body"
        >
          <form
            onSubmit={[Function]}
          >
            <div
              className="form-group"
            >
              <h6>
                <label
                  htmlFor="email"
                >
                  Email
                </label>
                <br />
              </h6>
              <input
                name="email"
                onChange={[Function]}
                type="email"
                value=""
              />
            </div>
            <div
              class="modal-footer form-group"
            >
              <button
                class="btn "
                type="submit"
              >
                Send Email
              </button>
            </div>
          </form>
        </div>
      </div>
    </div>
  </div>
</div>
`;
